src/constants/common.ts

```diff
@@ -1,6 +1,6 @@
 //TODO: url 수정
 //const BASE_URL = 'http://localhost:3000';
-const BASE_URL = 'https://52.79.247.31';
+const BASE_URL = 'https://3.37.123.52/';
```

Comment:
```suggestion
const BASE_URL = 'https://3.37.123.52';
```

해야될 것 같아요 다 /register 이런 식으로 해서!

Reply:
저거 있어도 잘 되더라고요?! 뭐지??
아무튼 지워두겠습니당